target/linux/ath79/dts/ar7242_meraki_mr12.dts

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+#include "ar7242.dtsi"
```

Comment:
I'd add an empty line before the include.

Reply:
Done.